modules/tinymce/src/core/main/ts/newline/InsertBlock.ts

```diff
@@ -147,12 +148,12 @@ const setForcedBlockAttrs = (editor: Editor, node: Element) => {
 // Wraps any text nodes or inline elements in the specified forced root block name
 const wrapSelfAndSiblingsInDefaultBlock = (editor: Editor, newBlockName: string, rng: Range, container: Node, offset: number) => {
   const dom = editor.dom;
-  const editableRoot = NewLineUtils.getEditableRoot(dom, container);
+  const editableRoot = CefUtils.getContentEditableRoot(dom.getRoot(), container) as Element;
```

Comment:
This cast should definitely not be done... this could potentially be nullable as the types mention so please don't pretend it's not.

Reply:
In this case it looks like we shouldn't even be getting this again here and we instead should be passing it down from where it's retrieved and checked in `insert`.

Reply:
Would a better solution be adding the original `NewLineUtils.getEditableRoot` function to CefUtils?

Reply:
Change has been reverted